test/integration/integration_test.go

```diff
@@ -279,31 +288,57 @@ func checkConfigKey(configPath string, condition string, keyPath string) error {
 
 func compareExpectedWithActualContains(expected string, actual string) error {
 	if !strings.Contains(actual, expected) {
-		return fmt.Errorf("Output did not match. Expected: %s, Actual: %s", expected, actual)
+		return fmt.Errorf("Output did not match. Expected: '%s', Actual: '%s'", expected, actual)
```

Comment:
I agree that there is no need for optimisation. I also agree that we don't really have to change it. I just said, _errors.New_ communicates better the intend imo. Even w/o knowing anything about Go, I can understand what _errors.New_ does. _fmt.Errorf_ is less obvious imo. Personally I had to look up the implementation/docs the first time I saw this function called. 

Also, in the production code we definitely use more _errors.New_. I have not seen many (if any)  _fmt.Errorf_.

Either way, happy to keep it.